src/snyk/cli/cliExecutable.ts

```diff
@@ -1,37 +1,69 @@
-import * as fs from 'fs/promises';
 import path from 'path';
-import { Platform } from '../common/platform';
-import { Checksum } from './checksum';
+import fs from 'fs/promises';
 import { CliSupportedPlatform } from './supportedPlatforms';
+import { Checksum } from './checksum';
+import { Platform } from '../common/platform';
 
-// TODO: This file is to be removed in VS Code + Language Server feature cleanup. We need to ensure all users have migrated to use CLI path that's set by the language server.
 export class CliExecutable {
-  // If values updated, `.vscodeignore` to be changed.
   public static filenameSuffixes: Record<CliSupportedPlatform, string> = {
     linux: 'snyk-linux',
-    win32: 'snyk-win.exe',
-    darwin: 'snyk-macos',
+    linux_alpine: 'snyk-alpine',
+    macos: 'snyk-macos',
+    macos_arm64: 'snyk-macos-arm64',
+    windows: 'snyk-win.exe',
```

Comment:
what about windows-arm64?

Reply:
agree.. we need to handle all these cases here..
Also port the changes to IntelliJ afterwards.